### PR TITLE
MicroK8s Windows Tutorial

### PR DESCRIPTION
**Pull request by ammarn911**
Created install-microk8s-on-windows.md

## Done

MicroK8s Tutorial for Windos PC

## QA

- Please review and test instructions.
- Please review tutorial layout on staging environment.

### CODE REVIEW

**Comment by ammarn911** [comment]
@ktsakalozos @caldav please review. Should be a quick one, it's identical to MicroK8s on Mac with Multipass. 

Please let me know your thoughts and if there's an issue. Thanks!

**Comment by evilnick** [comment]
@ktsakalozos have you looked over this?

File: tutorials/containers/install-microk8s-on-windows/install-microk8s-on-windows.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
+---
+id: install-microk8s-on-windows
+summary: Get a local Kubernetes on a Windows PC with Microk8s and Multipass.
```
Comment (caldav) [inline tutorials/containers/install-microk8s-on-windows/install-microk8s-on-windows.md]:
```suggestion
summary: Get a local Kubernetes on Windows with Microk8s and Multipass.
```

File: tutorials/containers/install-microk8s-on-windows/install-microk8s-on-windows.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+tags: kubernetes, beginner, multipass, container, microk8s
+difficulty: 3
+status: draft
+published: 2019-09-13
+author: Ammar Naqvi <ammar.naqvi@canonical.com>, Konstantinos Tsakalozos <kos.tsakalozos@canonical.com>
+feedback_url: https://github.com/ubuntu/microk8s/issues/
+
+---
+
+# Install MicroK8s on a Windows PC using Multipass
```
Comment (caldav) [inline tutorials/containers/install-microk8s-on-windows/install-microk8s-on-windows.md]:
```suggestion
# Install MicroK8s on Windows using Multipass
```

File: tutorials/containers/install-microk8s-on-windows/install-microk8s-on-windows.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+```
+
+## Using MicroK8s
+Duration: 2:00
+
+Open a shell in Multipass with a MicroK8s VM
+```bash
+multipass shell microk8s-vm
+```
+To execute a command without getting a shell, you can use multipass exec as in the example below:
```
Comment (caldav) [inline tutorials/containers/install-microk8s-on-windows/install-microk8s-on-windows.md]:
```suggestion
To execute a command without getting a shell, you can use `multipass exec` as in the example below:
```

**Pull request merged**